### DISCUSSION TITLE
Fixed order of emote groups

### DISCUSSION THREAD

**Discussion by CaptainFlint** [Chatterino/chatterino2#5289] [Ideas]
Each time I start Chatterino and open the channel Emotes window on the Subs tab, the emotes are displayed in groups (Tier 1, 2, 3, animated, follower emotes...). The problem is, the groups themselves are sorted completely randomly. Sometimes it's T3-T2-T1-animated-follower; other times T1-animated-follower-T2-T3-animated (not sure why, but there are several groups of animated emotes); other orderings also occur.

It would be nice to have fixed position for each group, so that they always displayed in the same order. It would let people remember the positions of the most used emotes and click them immediately, without searching. But this reordering is very off-putting (even though emotes are sorted the same within each group, it's the global position of the emote in the window is what's remembered).

P. S. I realize that when you change your sub tier and/or follower status, some of the groups may disappear/reappear, but that's cannot be avoided, so it's OK to skip some groups that are unavailable, as long as other groups keep their relative positions.

**Comment by pajlada** [comment]
There's some experimental work being done to fetch the emotes you can use in a different way (see https://github.com/Chatterino/chatterino2/pull/5239), I've added a comment to that effort to ensure this will work when that is being done

**Reply by CaptainFlint** [reply]
That's good to know, thank you!
